org/models.py

```diff
@@ -20,22 +20,13 @@ class Org (models.Model):
     def __str__(self):
         return self.name
 
-class PermissionSet(models.Model):
-    name = models.CharField(max_length=30, null=False, blank=False)
-    org = models.ForeignKey(Org,on_delete=models.CASCADE)
-    perm_obj = PermissionField()
-    
-    def __str__(self):
-        return f'{self.id}-{self.perm_obj.permissions_to_integer()}'
-
-
 
 class Group (models.Model):
     name = models.CharField(max_length = 30, null=False, blank=False)
     role = models.CharField(max_length = 200)
     invite_slug = models.SlugField(max_length = 40, unique=True)
     org = models.ForeignKey(Org,on_delete=models.CASCADE)
-    default_permission_set  = models.ForeignKey(PermissionSet, on_delete=models.CASCADE)
+    default_permissions = PermissionField()
```

Comment:
```suggestion
    perm_obj = PermissionField()
```
`perm_obj` will sound better in my opinion like `admin.perm_obj.permissions` vs `admin.default_permissions.permissions`. also since the permissions are fixed to the group no need the `default` tag anymore.

Reply:
Done 👍